packages/platform/atoms/event-types/atom-api-transformers/transformApiEventTypeForAtom.ts

```diff
@@ -194,7 +194,7 @@ function getLocations(locations: EventTypeOutput_2024_06_14["locations"]) {
         return displayLocationPublicly ? location : { ...location, address: "" };
       case "link":
         return displayLocationPublicly ? location : { ...location, link: "" };
-      case "phone":
+      case "userPhone":
```

Comment:
Good observation. That's correct. I think the `case "address"` is also wrong; it should be `case "inPerson"`